helpers.py

```diff
@@ -19,8 +19,10 @@ def mongo_to_dict(obj):
 
 
 def request_to_dict(request):
-    """Convert incoming flask requests into a dict"""
-    req_dict = request.values.to_dict()
+    """Convert incoming flask requests for objects into a dict"""
+    req_dict = request.values.to_dict(flat=True)
+    if request.is_json:
+        req_dict = request.get_json()  # get_dict returns python dictionary object
```

Comment:
this could be `req_dict.update(request.get_json())` but I'm still not sure what the best way to do this is - could requests contain json AND non-json data?